src/components/Book/Book.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable */
+import React from 'react';
+import { CircularProgressbar } from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
+import './Book.css'
+
+function Book(props) {
+  const { title, author, category, progress, chapter } = props;
+  return (
+    <div className="grid-container">
+      <div className="grid-item items-1">
+        <p>{category}</p>
+        <h2>{title}</h2>
+        <p className="author">{author}</p>
+      </div>
+      <div className="grid-item items-2">
+        <div style={{ width: 100, height: 100}}>
+        <CircularProgressbar value={progress} />
+        </div>
+        <div className="progress">
+        <p className="percent">{progress}%</p>
+        <p className="completed">Completed</p>
+        </div>
+      </div>
+      <div className="grid-item vl"></div>
+      <div className="grid-item items-3">
+        <p className="current">Current Progress</p>
+        <p>{chapter}</p>
+        <button type="button">Update Progress</button>
+      </div>
+    </div>
```

Comment:
- [ ] in the `Book` component according to the requirements there should be a button to remove a book. the functionality for this button should not be implemented yet. please make this change thanks.

Reply:
Thank you for your review. I have fixed this issue.
![Screenshot from 2022-03-26 16-39-30](https://user-images.githubusercontent.com/60083437/160235817-1b2c2741-06a4-46de-acfe-4051032e5fc8.png)